<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>~~ Enter Password ~~</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="header">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enter Password:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="cssClass" stdset="0">
      <string>neutral bold</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClickableLineEdit" name="password_field">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200000</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="cssClass" stdset="0">
      <string>bold secondary</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="PB2">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="PB1">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="PB0">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="PB9">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="PB4">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="PB3">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="PB8">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="PB7">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="PB6">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="PB5">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QPushButton" name="enter_button">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="cssClass" stdset="0">
        <string>btn-inverted</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="PBdel">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>del</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="cancel_button">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="cssClass" stdset="0">
        <string>btn-inverted</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src.ui_elements.clickablelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
